### PR TITLE
Feature1

### PR DESCRIPTION
**Pull request by yaseeen96**
add feature 1

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,5 +6,8 @@
   </head>
   <body>
     <h1>Git testing</h1>
+    <h2>I need this feature</h2>
+    <h3>I need this feature as well</h3>
+    <a href="feature 1/feature1.html">this is feature 1</a>
```
Comment (mdmusaibali) [inline index.html:11]: Do this better

**Review by mdmusaibali — COMMENTED** [review]
Do the changes

File: index.html (hunk 1 above)
Comment (mdmusaibali) [reply index.html:11]: Add new h3

Comment (mdmusaibali) [reply index.html:11]: sahi se kar ssale

Comment (mdmusaibali) [reply index.html:11]: try again

**Review by mdmusaibali — COMMENTED** [review]
test

File: index.html (hunk 1 above)
Comment (yaseeen96) [reply index.html:11]: ye le saaale

**Pull request merged**